components/match2/wikis/clashofclans/match_group_input_custom.lua

```diff
@@ -112,6 +112,11 @@ end
 --
 -- map related functions
 --
+---@param map table
+---@return string?
+function MapFunctions.getMapName(map)
+	return nil
+end
```

Comment:
longterm if we sep input and processed map data this function can be dropped again^^

Reply:
yup